opal/mca/rcache/base/rcache_base_vma_tree.c

```diff
@@ -131,6 +132,17 @@ mca_rcache_base_vma_item_t *mca_rcache_base_vma_new (mca_rcache_base_vma_module_
     return vma_item;
 }
 
+/* NTH: this function MUST not allocate or deallocate memory */
```

Comment:
Oh wait. Sorry about that. The comment was on a function that was deleted before the final version of the patch. Will eliminate the incorrect comment.